test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -328,6 +332,26 @@ UCS_TEST_P(test_ucp_sockaddr, err_handle) {
 
 UCP_INSTANTIATE_ALL_TEST_CASE(test_ucp_sockaddr)
 
+
+class test_ucp_sockaddr_with_wakeup : public test_ucp_sockaddr {
+public:
+
+    static ucp_params_t get_ctx_params() {
+        ucp_params_t params = ucp_test::get_ctx_params();
```

Comment:
minor: can call test_ucp_sockaddr::get_ctx_params() instead of ucp_test::get_ctx_params(), then the only thing need to do is to `params.features  |= UCP_FEATURE_WAKEUP;`